src/libs/ReportUtils.ts

```diff
@@ -1801,7 +1801,7 @@ function hasNonReimbursableTransactions(iouReportID: string | undefined): boolea
     return transactions.filter((transaction) => transaction.reimbursable === false).length > 0;
 }
 
-function getMoneyRequestReimbursableTotal(report: OnyxEntry<Report>, allReportsDict: OnyxCollection<Report> = null): number {
+function getMoneyRequestTotal(report: OnyxEntry<Report>, allReportsDict: OnyxCollection<Report> = null): number {
```

Comment:
I renamed this method because the previous name was misleading. It said it was getting the reimbursable total, but it in fact just returned the report total (nonReimbursable + reimbursable)